edi_exchange_template_oca/migrations/18.0.1.1.0/post-migrate.py

```diff
@@ -0,0 +1,60 @@
+# Copyright 2025 Camptocamp SA (http://www.camptocamp.com)
```

Comment:
@yankinmax  as we decided to drop the deprecated implementation, please change this. 
We should simply move this code to the pre-migrate step but we should create via SQL

* the column `output_template_id` col on the type
* the table `edi_exchange_template_type_rel` 

and set their values via SQL.
Keep in mind that the new fields won't be available at ORM level :wink: 

.Then you can trash the deprecated code + deprecated tests/

Reply:
I updated this PR that way to be sure it won't break any build while we're handling the deprecated code.
I'll do the job in coming weeks